README.md

```diff
@@ -1,97 +1,51 @@
-# Quick Start
```

Comment:
We are not able to change the name of this document because it is also the Quick Start in the docs.